src/imputation/MoR.py

```diff
@@ -29,7 +29,7 @@ def run_mor(df, backdata, impute_vars):
     carried_forwards_df = carry_forwards(to_impute_df, backdata, impute_vars)
 
     # TODO Remove the `XXX_prev` columns (left in for QA)
-    return pd.concat([remainder_df, carried_forwards_df])
+    return pd.concat([remainder_df, carried_forwards_df]).reset_index()
```

Comment:
@AnneONS just a heads up this is the bugfix I added. I think it was caused by us not doing anything with the index when we performed the merge.

Reply:
Great, thanks.

Reply:
@coatet This should solve the index issue

Reply:
@dpurches  is there a way of solving the problem without it creating an extra "index" column? This comes in the output csv even though we are writing with "index=False"

Reply:
Don't worry, I googled it :-) it's reset_index(drop=True)